src/webgpu/api/validation/image_copy/texture_related.spec.ts

```diff
@@ -89,31 +89,38 @@ g.test('usage')
   });
 
 g.test('sample_count')
-  .desc(`Multisampled textures cannot be copied.`)
+  .desc(`It must be a full copy if the texture is multisampled.`)
   .params(u =>
     u //
       .combine('method', kImageCopyTypes)
       .beginSubcases()
       .combine('sampleCount', [1, 4])
+      .combine('copyWidthRatio', [1.0, 0.5])
```

Comment:
nit: we could have something like `copyWidthModifier: [0, -1]` and same for height to check exactly the condition.